4/Dockerfile

```diff
@@ -15,10 +15,9 @@ RUN set -ex \
     56730D5401028683275BD23C23EFEFE93C4CFFFE \
     77984A986EBC2AA786BC0F66B01FBB92821C587A \
   ; do \
-    gpg --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" || \
-    gpg --keyserver hkp://keyserver.pgp.com:80 --recv-keys "$key" || \
+    gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
```

Comment:
we should not reorder this one here, as it's not mentioned

Reply:
I updated my PR comments. I deliberately put `p80.pool.sks-keyservers.net:80` as the first server to try before `hkp://ipv4.pool.sks-keyservers.net`.

Should we try `hkp://ipv4.pool.sks-keyservers.net` first?

Reply:
I'm not sure what's our current ordering, popularity, reliability, security or ... some others, maybe we could figure it out this time?
/cc @nodejs/docker

Reply:
I guess whichever is the most reliable should be first, which in theory should be the `*.pool.sks-keyservers.net` servers.

`hkp://p80.pool.sks-keyservers.net:80` was chosen to force port 80 for situations where users are behind a proxy. We added `hkp://ipv4.pool.sks-keyservers.net` to address an issue with GPG's dirmngr: https://github.com/nodejs/docker-node/issues/380#issuecomment-306451705

FWIW, adding those two servers, at least as far as I can tell, seems to have helped with a lot of the failures we were getting in the Alpine images when fetching gpg keys. It probably wouldn't hurt to add `ha.pool.sks-keyservers.net` back, maybe as the first server to try, then fall back to `hkp://p80.pool.sks-keyservers.net:80` etc. I'd rather tackle that in a new PR though :)

Reply:
Understood now, thanks, I'm fine with it, @SimenB what do you think?